pkg/util/test/util.go

```diff
@@ -3064,6 +3078,70 @@ func ValidateTelemetryV2Enabled(pxImageList map[string]string, cluster *corev1.S
 	return nil
 }
 
+func validatePxAuthOnPxNodes(pxAuthShouldBeEnabled bool, cluster *corev1.StorageCluster) error {
+	listOptions := map[string]string{"name": "portworx"}
+	cmd := "pxctl status"
+	cmdWithPxAuthToken := ""
+
+	logrus.Infof("Run pxctl status on all PX nodes to determine if PX Auth is enabled/disabled properly")
+	t := func() (interface{}, bool, error) {
+		if cluster.Spec.Security != nil && cluster.Spec.Security.Enabled {
```

Comment:
we already have `pxAuthShouldBeEnabled` as an input, so just use `if pxAuthShouldBeEnabled`?

Reply:
Done